Parse/Parse.h

```diff
@@ -67,6 +68,22 @@ NS_ASSUME_NONNULL_BEGIN
 + (void)setApplicationId:(NSString *)applicationId clientKey:(NSString *)clientKey;
```

Comment:
I never realized that iOS uses `set` instead of `initialize`. I do like moving towards `initialize` since it's not exactly a setter, but is it weird to have it be named differently?

Also, if we keep `initialize`, we should probably update the abstract of it from `Sets the configuration to be used for parse` to `Initialize Parse with a configuration` or something